extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -642,9 +644,14 @@ private void handleHibernateORMWithNoPersistenceXml(
                                 fetchSize.toString()));
 
                 // Logging
-                if (hibernateConfig.log.sql) {
-                    desc.getProperties().setProperty(AvailableSettings.SHOW_SQL, "true");
-                    desc.getProperties().setProperty(AvailableSettings.FORMAT_SQL, "true");
+                if (hibernateConfig.log.sql.isPresent()) {
+                    HibernateOrmConfig.HibernateOrmConfigLog.HibernateOrmConfigLogSqlValue value = hibernateConfig.log.sql
+                            .get();
+                    if (value.ordinal() >= HibernateOrmConfig.HibernateOrmConfigLog.HibernateOrmConfigLogSqlValue.TRUE
```

Comment:
Please don't rely on the ordinal, it's prone to errors in case someone re-orders the enum constants down the road. I think simply checking for the two enum constants is good enough; or, if you'd want something more generic, add methods to the enum such as `boolean logsSql()` (returning true for TRUE and EXTENDED) and `boolean logsParameters()`.